src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -161,6 +162,14 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   ngOnInit() {
+    this.encryptForm = this.formBuilder.group({
+      'password': ['', [Validators.required]],
+      'confirmPwd': ['', [Validators.required]],
+      'passwordHint': ['', [Validators.required]]
```

Comment:
@Waseem826 I think the password hint should not be required.

Reply:
Fixed